client/utils/formatter.js

```diff
@@ -19,3 +21,14 @@ export const capitalizeEachWord = (
 
     return words.join(joinChar);
 };
+
+export const convertDateToString = (date, format = 'DD-MM-YYYY') => {
+    if (!date) return '';
+    return moment(date)
+        .format(format)
+        .toString();
```

Comment:
Maybe `toString()` is redundant? I think `format` returns a string (?)

Reply:
Seems to be the case. Good catch!